src/freenas/usr/local/sbin/ctdb_glfs_lock

```diff
@@ -40,7 +40,18 @@ class GlusterConn:
         return
 
     def take_reclock(self):
-        lock_file = self.glusterhdl.get_root_handle().lookup(self.reclock_path)
+        try:
+            lock_file = self.glusterhdl.get_root_handle().lookup(self.reclock_path)
+        except pyglfs.GLFSError as e:
+            if e.errno != errno.ENOENT:
+                raise
+
+            lock_file = self.glusterhdl.get_root_handle().create(
```

Comment:
Does `.create()` return the same as `.lookup()`?

Reply:
Yup. Both return a new glusterfs object handle.

```
PyDoc_STRVAR(py_glfs_obj_create__doc__,
"create(path, flags, stat=False, symlink_follow=False, mode=0o644)\n"
"--\n\n"
"Create new GLFS object (file) by path.\n\n"
"Parameters\n"
"----------\n"
"path : str\n"
"    Path of object relative to this handle.\n"
"flags : int\n"
"    open(2) flags to use to open the handle.\n"
"stat : bool, optional, default=True\n"
"    Retrieve stat information for object while performing create.\n"
"symlink_follow: bool, optional, default=True\n"
"    Follow symlinks while performing create.\n\n"
"mode : bool, optional, default=0o644\n"
"    Permissions to set on newly created directory.\n\n"
"Returns\n"
"-------\n"
"pyglfs.ObjectHandle\n"
"    New GLFS handle\n"
);
```

vs

```
PyDoc_STRVAR(py_glfs_obj_lookup__doc__,
"lookup(path, stat=False, symlink_follow=False)\n"
"--\n\n"
"Lookup existing GLFS object by path.\n\n"
"Parameters\n"
"----------\n"
"path : str\n"
"    Path of object relative to this handle.\n"
"stat : bool, optional, default=True\n"
"    Retrieve stat information for object while performing lookup.\n"
"symlink_follow: bool, optional, default=True\n"
"    Follow symlinks while performing lookup.\n\n"
"Returns\n"
"-------\n"
"pyglfs.ObjectHandle\n"
"    New GLFS handle\n"
);
```